packages/dashboard/src/components/internalDashboard/index.tsx

```diff
@@ -111,18 +114,25 @@ const InternalDashboard: React.FC<InternalDashboardProperties> = ({ onSave, edit
   };
 
   const deleteWidgets = () => {
+    setVisible(true);
+  };
+
+  const onDelete = () => {
     dispatch(
       onDeleteWidgetsAction({
         widgets: selectedWidgets,
       })
     );
+    setVisible(false);
   };
 
   /**
    * setup keyboard shortcuts for actions
    */
   useKeyboardShortcuts();
 
+  useKeyPress('backspace, del', { filter: keyPressFilter, callback: deleteWidgets });
```

Comment:
I think this is already a part of keyboard shortcuts.

Reply:
resolved I moved back this to keyboard shortcuts